### ISSUE TITLE
Conebeam projection using torch.tensor

### ISSUE THREAD

**Issue by yongjinColinChoi** [astra-toolbox/astra-toolbox#387]
I have a question about Conebeam projection using astra-library.

I have a CT volume, and I want to Project it using conebeam geometry like id, projdata = astra.create_sino3d_gpu(cube_id, proj_geom, vol_geom).

here is my question. I want Input CT volume's data type to be torch.tensor and requires_grad = True, so that after conebeam projection I can use torch.autograd. 

Is there any method to project 3D conebeam geometry using input tensors and can track gradient during projection?
